src/nvim/screen.c

```diff
@@ -958,7 +958,7 @@ static void win_update(win_T *wp, Providers *providers)
   if ((type == VALID || type == SOME_VALID
        || type == INVERTED || type == INVERTED_ALL)
       && !wp->w_botfill && !wp->w_old_botfill
-      ) {
+  ) {
```

Comment:
ideally, this should be joined with the previous line instead.

Reply:
fixed